src/univariate/continuous/triangular.jl

```diff
@@ -35,7 +35,7 @@ struct TriangularDist{T<:Real} <: ContinuousUnivariateDistribution
 end
 
 function TriangularDist(a::T, b::T, c::T; check_args::Bool=true) where {T <: Real}
-    @check_args TriangularDist (a <= c <= b)
+    @check_args(TriangularDist, (a <= c <= b, "must have a <= c <= b"), (a < b, "must have a < b"))
```

Comment:
Is this actually needed? A few distributions also support the degenerate case (e.g., `Normal(m, 0)`), so I would not want to add this restrictions if it is not required.

I would also suggest using the default error messages, IMO the manual ones do not add more explanations:
```suggestion
    @check_args TriangularDist (a <= c <= b) (a < b)
```

Reply:
I'm good with default error messages and have accepted your suggested change.  To me, there's no point in allowing someone to define a trivial/degenerate distribution.  I can't think of a reason to intentionally define it.